### DISCUSSION TITLE
Can it be replaced with left top as the reference point when reading svg?

### DISCUSSION THREAD

**Discussion by sanheng** [fabricjs/fabric.js#7325] [Q&A]
When reading the svg, the clippath object always takes the top left as the center point, can it be replaced with left top as the reference point?

**Comment by asturur** [comment]
@sanheng is unclear to me what are you asking.
please try to be more clear.

**Reply by sanheng** [reply]
When I set the top and left of the image to 0, I found that it is based on the center of the background. I wonder if I can set it to be based on the background top left
And image.clippath is also based on the center point of the parent element

![image](https://user-images.githubusercontent.com/13793256/130473276-39ce4e97-196b-464c-b4f6-5713ad16548f.png)
